src/index.js

```diff
@@ -1,11 +1,11 @@
 const BotmeterLoggerBotbuilder = require('./botmeter_logger_botbuilder');
 const BotmeterLoggerMessenger = require('./botmeter_logger_messenger');
 const BotmeterLoggerBotfuel = require('./botmeter_logger_botfuel');
+const BotmeterLogger = require('./botmeter_logger');
 
-module.exports = auth => {
-  return ({
-    botbuilder: new BotmeterLoggerBotbuilder(auth),
-    messenger: new BotmeterLoggerMessenger(auth),
-    botfuel: new BotmeterLoggerBotfuel(auth),
-  });
-}
+module.exports = auth => ({
+  botbuilder: new BotmeterLoggerBotbuilder(auth),
+  messenger: new BotmeterLoggerMessenger(auth),
+  botfuel: new BotmeterLoggerBotfuel(auth),
+  generic: new BotmeterLogger(auth),
```

Comment:
this is strange, we initiate 4 objects to use only one. Maybe we shouldn't care for now, what do you think?

Reply:
Yes I agree this is strange, the objects should only be initialized if needed, this should be fixed later